lib/workload/stateless/stacks/sample-sheet-check/sample-sheet-check-lambda/handler.py

```diff
@@ -49,12 +49,19 @@ def lambda_handler(event, context):
                               v2_sample_sheet='')
         response = construct_response(status_code=400, body=body, origin=origin)
         return response
-
     multipart_content = {}
     for part in msg.get_payload():
         multipart_content[part.get_param(
             'name', header='content-disposition')] = part.get_payload(decode=True)
 
+    key_list = list(multipart_content.keys())
+    if "file"not in key_list or "logLevel" not in key_list:
```

Comment:
Should be a space between 'file' and 'not'

Reply:
Updated, although not sure why PyCharm didn't complain this.